### PR TITLE
Cove renaming pt2

### PR DESCRIPTION
**Pull request by atishp04**
Based on #27. 

FYI: I have not changed any references in attestation section to avoid conflicts for @sameo's ongoing PR #17

### CODE REVIEW

**Comment by atishp04** [comment]
@rsahita : Do you have a raw file for the figure 4 (Memory Tracking for Confidential PMA) ?
This series replaces the "AP-TEE mode" with "Confidential mode" based on your CoVE paper. The paper have the updated figure. So I thought I would check with you first before using my shitty photoshop/drawing skills :)

**Comment by atishp04** [comment]
Should we remove the references to T[G/H]-ABI & TEEI ? I don't see any value add those acronyms. 
We can just replace TEEI with CoVE Interface & T[G/H]-ABI with COV[H/G]. The figures also need to be updated.

**Comment by atishp04** [comment]
@rsahita Updated the PR as per our discussion. The diagram is still old as I don't have edit access to the source of the image.

File: specification/glossary.adoc [hunk 1]
```diff
@@ -45,20 +45,20 @@ also used as synonyms of AP-TEE in this document.
 | Confidential Computing | The protection of data in use by performing 
 computation in a Hardware-based TEE.
 
-| TVM | TEE or Confidential VM - A VM instantiation of an AP-TEE workload
+| TVM | TEE or Confidential VM - A VM instantiation of an confidential workload
```
Comment (rsahita) [inline specification/glossary.adoc:48]:
```suggestion
| TVM | TEE VM (TVM) or Confidential VM - A VM instantiation of a Confidential VM Extension workload
```

File: specification/glossary.adoc [hunk 2]
```diff
@@ -45,20 +45,20 @@ also used as synonyms of AP-TEE in this document.
...
 
 | Confidential application or library | A user-mode application or 
 library instantiation in a TVM. The user-mode application may be supported 
 via a trusted runtime. The user-mode library may be hosted by a surrogate 
 process runtime.
 
 | Attestation | The process by which a relying party can assess the 
-security posture of the AP-TEE workload based on verifying a set of 
+security posture of the confidential workload based on verifying a set of
```
Comment (rsahita) [inline specification/glossary.adoc:56]:
```suggestion
security posture of the confidential VM workload based on verifying a set of 
```

File: specification/overview.adoc [hunk 3]
```diff
@@ -117,13 +117,13 @@ guest OS runtime may be used; whereas in the VM TEE case, an enlightened
...
 
 image:img_1.png[]
 Figure 2: TEE TCB for application workloads (hosted via a TVM)
 
 The detailed architecture is described in the Section 
 <<refarch>>. Note that the architecture described
 above may have various implementations, however the goal of this specification 
-is to propose a reference architecture and ratify a normative TEEI as a RISC-V non-ISA 
-specification.
+is to propose a reference architecture and ratify a normative COVE as a RISC-V
```
Comment (rsahita) [inline specification/overview.adoc:128]:
```suggestion
is to propose a reference architecture and ratify a normative COVE ABI as a RISC-V
```

**Review by rsahita — APPROVED** [review]

**Pull request merged**